src/components/alert.js

```diff
@@ -1,43 +1,36 @@
-import { appRouter } from './router/appRouter';
-import browser from '../scripts/browser';
-import dialog from './dialog/dialog';
-import globalize from '../scripts/globalize';
-
-function useNativeAlert() {
-    // webOS seems to block modals
-    // Tizen 2.x seems to block modals
-    return !browser.web0s
-            && !(browser.tizenVersion && browser.tizenVersion < 3)
-            && browser.tv
-            && window.alert;
-}
+import { appRouter } from "./router/appRouter";
```

Comment:
Sorry for the delay! The eslint issues listed here need fixed. You should be able to run `npx eslint --fix` locally to fix them automatically. Thanks!